thunderbird-development/building-thunderbird/linux-build-prerequisites.md

```diff
@@ -50,9 +106,13 @@ Please choose the version of Firefox you want to build:
 
 Please choose option 2 to proceed with a successful build.
 
-This action will install all the libraries and dependencies necessary to build Thunderbird locally.
+This action should install all the remaining libraries and dependencies necessary to build Thunderbird locally.
+
+### Create `mozconfig` file
+
+This step will need to be performed if you manually checked out the code and performed the bootstrap, and it will covered in the next section you follow, [Building Thunderbire](./#build-configuration).
```

Comment:
This section should be before the "Mach Bootstrap" section otherwise users will build Firefox.

Reply:
you're absolutely right.